### ISSUE TITLE
Update R Version Installation Guidance

### ISSUE THREAD

**Issue by mblue9** [carpentries-incubator/bioc-project#79]
Hi Kevin,

Vince (@vjcitn) noticed the docs suggest installing R 4.1.x [here](https://carpentries-incubator.github.io/bioc-project/) but had difficulty finding how to change this advice via 'Edit in GitHub'. Considering R's frequent updates, perhaps directly linking to the Bioconductor R version advice could make access easier?

This isn't urgent, but I'm interested in your thoughts on future-proofing this guidance.

Thanks!
Maria

**Comment by kevinrue** [comment]
Yes, good point, I don't think we want to update the lesson annually.

To be honest, having those instructions elsewhere  and linking to them just displaces the issue, as that "other place" will have to be kept up to date itself each year.

However, it the Bioconductor website already has such advice, then yes, that seems the best place to link to. And if it doesn't have that page yet, it sure sounds like a key addition for new users.
